centreon/src/App/ActivityLogging/Domain/Aggregate/ActivityLog.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * Copyright 2005 - 2025 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace App\ActivityLogging\Domain\Aggregate;
+
+use App\Shared\Domain\Exception\MissingIdException;
+
+final class ActivityLog
+{
+    public function __construct(
+        private ?ActivityLogId $id,
+        public readonly ActionEnum $action,
+        public readonly Actor $actor,
```

Comment:
Any reason to have this two level VO Actor + ActorId without anything else than ActorId in Actor?

Reply:
I'm ok with it